### PR TITLE
update docs for install and migrations

### PR DESCRIPTION
**Pull request by zypro**
This PR solves #3 and #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +111,45 @@ VALIDATOR_CLASSY = Validator(
 )
 ```
 
+## Migrations
+
+If you need to migrate from TextField or CharField to the MarkdownField you need to migrate the stored `text` also in the `rendered_text` field.
+Update your auto-created migration fiele and add the method below. 
+Use a method to `save()` every instance of your model once after the migrations, so the text will be copied into the `text_rendered` field correctly.
+
+```
```
Comment (baseplate-admin) [inline README.md]: This should be 

```
   ```python
```

Comment (zypro) [reply README.md]: I'll update it.

**Comment by cyzanfar** [comment]
This should be merged. I just spent quite some time figuring out why it wasn't working. Thanks @zypro!

**Review by benjaoming — APPROVED** [review]
Thanks! This is a great addition! The README is slowly turning into a real documentation project :)

**Comment by zypro** [comment]
> Thanks! This is a great addition! The README is slowly turning into a real documentation project :)

great that you are taking care of this project now.

**Comment by benjaoming** [comment]
Community already did most of the work, and @dmptrluke is still around, so this is gonna be great :heart:

**Pull request merged**